Traceback (most recent call last):
  File "mpy_cross/SPB_2021a.py", line 7, in <module>
SyntaxError: 'break' within finally
